README.md

```diff
@@ -219,6 +220,19 @@ await client.logon();
 await client.logoff();
 ```
 
+## onSessionExpiration callback
+The onSessionExpiration is callback with the sdk client as parameter, it is called when the ACC session is expired.
```

Comment:
The onSessionExpiration is **a** callback with the sdk client as parameter, it is called when the ACC session is expired.